src/test/kotlin/uk/gov/justice/digital/hmpps/courtregister/resource/CourtResourceTest.kt

```diff
@@ -246,7 +247,7 @@ class CourtResourceTest : IntegrationTest() {
       val building1 = Building(
         id = 1,
         court = court,
-        subCode = "SUBT111",
+        subCode = "SUBT11",
```

Comment:
test data incorrect had a sub-code too long, so this corrected all over the place